tests/test_uploads.py

```diff
@@ -80,6 +80,7 @@ def testGet(self):
         self.assertEqual(rv.status_code, 404)
 
     def testRegularUploads(self):
+        '''Test that regular file should not be possible to upload.'''
         # Loop over some files and the status codes that we are expecting
         # Basically it should never be possible to upload a "regular" file.
```

Comment:
Can you remove this comment? It's just wrong. Possibly leftover from an earlier version of this test.